operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -878,6 +878,19 @@ def generateAndValidate(self):
     if not os.path.exists(self.env.CUSTOM_SITCFG_PATH):
       return
 
+    # We expect the user to include a 'version.txt' file in their situational
+    # config directory.
+    #
+    # That file is expected to contain '2.0'
+    #
+    versionPath=os.path.join(self.env.CUSTOM_SITCFG_PATH,"version.txt")
+    if not os.path.exists(versionPath):
+        self.env.addError("Error, Required file, "+versionPath+", does not exist")
+    else:
+        version=self.env.readFile(versionPath).strip()
+        if not version == "2.0":
+            self.env.addError("Error, "+versionPath+" does not have the value of '2.0'.")
```

Comment:
Can you add a quoted something to this error message that shows the first 100 characters of the invalid version?  (100 is arbitrary - just some value to prevent dumping too much if it's a huge file).     

Also, please add single quotes around 'versionPath' in the error messages.

Reply:
not sure I understand your second comment... what is the point of quotes around versionPath?